test/integration/integration.h

```diff
@@ -160,6 +155,15 @@ class BaseIntegrationTest : Logger::Loggable<Logger::Id::testing> {
                                                Http::CodecClient::Type type);
   IntegrationTcpClientPtr makeTcpConnection(uint32_t port);
 
+  // Test-wide port map.
```

Comment:
That's correct. Each individual `_test.cc` becomes an independent test target that is run in a separate process, and these may run in parallel. Within a `_test.cc`, gtest will run all tests sequentially in the same process.